thoth/digests_fetcher/python/fetch.py

```diff
@@ -38,13 +45,19 @@ def fetch(self, package_name: str, package_version: str) -> dict:
             "Fetching digests for package %r in version %r from %r",
             package_name,
             package_version,
-            self.source.url,
+            self.source.url
         )
 
         # TODO: implement
+        files = self.source.get_package_hashes(package_name, package_version, True)
+        # files = self.source.get_package_hashes(package_name, package_version)
+
+        # with open('files.json', 'w') as outfile:  
+            # json.dump(files, outfile)
+
```

Comment:
Please remove code which is not used and there is no reason to keep it in the repo.